core/domain/app_feedback_report_domain_test.py

```diff
@@ -199,6 +240,23 @@ def test_report_web_platform_validation_fails(self) -> None:
             'Domain objects for web reports have not been implemented yet.'):
             self.web_report_obj.validate()
 
+    def test_raise_not_implemented_error_for_web_platform_feedback(
+        self
+        ) -> None:
+        REPORT_JSON['platform_type'] = 'invalid_platform_type'
+        with self.assertRaisesRegex(
+            NotImplementedError,
+            'Domain objects for web reports must be implemented.'):
+            self.web_report_obj.from_submitted_feedback_dict(REPORT_JSON)
+
+    def test_create_web_report_with_submitted_feedback_returns_report(
```

Comment:
``test_create_web_report_with_submitted_feedback_returns_report`` -> ``test_web_report_creation_from_submitted_feedback``

Reply:
changed `test_create_web_report_with_submitted_feedback_returns_report` ->`test_android_report_creation_from_submitted_feedback` as it test creation of android feedback report.